### ISSUE TITLE
Please help i cant open EggCrack!!

### ISSUE THREAD

**Issue by Skyapplesauce** [Manevolent/EggCrack#33]
what is going on? I tried deleting the .sig file and it STILL does not work!!

**Comment by Herobrinekill15** [comment]
i cant get it open

**Comment by SavageKiller13** [comment]
have you placed the eggcrack shortcut with the right things?

**Comment by Beyarz** [comment]
This worked for me when I could not open EggCrack.jar.
1) Create a text file.
2) Type this "java -jar EggCrack.jar" without the quotes.
3) save it as run.bat or what ever you want.
4) run the bat file.

**Comment by Irvingchill** [comment]
@Beyarz what do you mean? Like Rename the text file tht And Change the extension or type it inside then save it as run.bat. How do you save it as run.bat im Dumb?

**Comment by Beyarz** [comment]
Yes, change the extension to .bat and run it after you're done with the second instruction.
